src/styles/helpers/pxToRem.tsx

```diff
@@ -0,0 +1,6 @@
+const BASE_FONT_SIZE_PX = 16;
```

Comment:
this should be a .ts file not .tsx as it contains no jsx. Please make it so and for all of the other files which don't contain jsx.